src/main/resources/css/style.css

```
@@ -0,0 +1,59 @@
+
+/*
+styles.css
+ */
+.root {
+    -fx-background-color:
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)